reference-adapters/mukta-ifix-adapter/src/main/java/org/egov/works/mukta/adapter/enrichment/PaymentInstructionEnrichment.java

```diff
@@ -353,6 +353,7 @@ public PaymentInstruction getPaymentInstructionFromDisbursement(Disbursement dis
         HashMap<String,List<Disbursement>> beneficiaryDisbursementMap = getBeneficiaryDisbursementMap(disbursement);
         for(Map.Entry<String,List<Disbursement>> entry: beneficiaryDisbursementMap.entrySet()){
             String beneficiaryId = entry.getKey();
+            BigDecimal totalAmountForBenef = BigDecimal.ZERO;
```

Comment:
the variable can be renamed to totalAmountForBeneficiary itself instead of using a shorthand